src/client/apps/edit/components/admin/test/components/article/article_authors.test.js

```diff
@@ -83,6 +83,12 @@ describe('ArticleAuthors', () => {
     expect(component.find(AutocompleteListMetaphysics).length).toBe(1)
   })
 
+  it('Does not render contributing authors if layout is news', () => {
+    props.article.layout = 'news'
+    const component = getWrapper(props)
+    expect(component.find(AutocompleteListMetaphysics).length).toBeFalsy()
```

Comment:
One totally unrelated thought that I had from your presentation today is that the `<AutocompleteListMetaphysics />` component could possibly be collapsed into just `<AutocompleteList />` with a datasource prop:

```javascript
<AutocompleteList
  dataSource={{
    metaphysics: {
      field: 'contributing_authors',
      label: 'Contributing Authors',
      model: 'foo'
    }
  }}
/>
```